tests/test_ml_streaming/scenarios/scenario2.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
+from sklearn.metrics import mean_squared_error
+from sklearn.linear_model import LinearRegression
+from pymilo.streaming.pymilo_client import PymiloClient
+from pymilo.utils.data_exporter import prepare_simple_regression_datasets
+
+
+def scenario2():
+    # 1. create model in local
+    # 2. upload model to server
+    # 3. train model in server
+    # 4. calculate mse in server
+    # 5. download model to local
+    # 6. calculate mse in local
+
+
+    # 1.
+    x_train, y_train, x_test, y_test = prepare_simple_regression_datasets()
+    linear_regression = LinearRegression()
+    client = PymiloClient(model=linear_regression, mode="LOCAL")
+
+    # 2.
+    client.upload()
+
+    # 3.
+    client.toggle_mode()
```

Comment:
I don't like the `toggle_mode` design. Finding out the current `mode` of a client connection, user should go through the code and count the number of toggles.

Reply:
I agree, please drop your suggestion. I will think about it too.

Reply:
having a `mode` argument which sets to `local` as default.

Reply:
hmmm, so your suggestion is that we keep this syntax and add an argument named mode?
like this:
`client.toggle_mode(mode="DELEGATE")`

Reply:
FYI:
we do have an attribute named `_mode` in the `PyMiloClient` too.

Reply:
No I meant `client.predict(..., mode='LOCAL')`